src/vma/util/agent.cpp

```diff
@@ -64,13 +64,15 @@
 	} while (0)
 
 /* Print user notification */
-#define output_warn() \
-	vlog_printf(VLOG_DEBUG, "Peer notification functionality is not active.\n"); \
-	vlog_printf(VLOG_DEBUG, "Check daemon state\n");
-
 #define output_fatal() \
-	vlog_printf(VLOG_DEBUG, "Peer notification functionality is not supported.\n"); \
-	vlog_printf(VLOG_DEBUG, "Increase output level to see a reason\n");
+	do { \
+		vlog_levels_t _level = (mce_sys_var::HYPER_MSHV == safe_mce_sys().hypervisor ?          \
+						VLOG_WARNING : VLOG_DEBUG);                                             \
+		vlog_printf(_level, "*************************************************************\n"); \
+		vlog_printf(_level, "* Can not establish connection with the daemon.             *\n"); \
+		vlog_printf(_level, "* Increase verbose level and check the daemon state.        *\n"); \
+		vlog_printf(_level, "*************************************************************\n"); \
+	} while (0)
```

Comment:
I am wondering if this message should appear at startup or during the creation of the first TCP socket... what do you think ?
Why do you suggest to increase the verbose level ?
why not specify how to enable the demon ?

Reply:
done